src/App.js

```diff
@@ -92,5 +124,5 @@ const StyledHeadline = styled.h2`
   grid-column: 1/3;
   margin-top: 100px;
   font-weight: 300;
-  color: #4BDB80;
+  color: #4bdb80;
```

Comment:
You could define your colors in your globalstyles, so it would be easier to use and the code would be a little easier to read.